persistence-api/pom.xml

```diff
@@ -16,6 +16,17 @@
     </repository>
   </repositories>
   <dependencies>
+    <dependency>
+      <groupId>org.apache.cassandra</groupId>
+      <artifactId>cassandra-all</artifactId>
+      <version>4.0-alpha4</version>
```

Comment:
This is required by to use C*'s `org.apache.cassandra.metrics.DefaultNameFactory`, similar to how we are using in the `cql` module.

I'll create a ticket to get rid of this dependency and `cql`'s.